scan.go

```diff
@@ -155,12 +175,21 @@ func (c *ScanCmd) Execute(args []string) error {
 	return nil
 }
 
-func scan(pkgPatterns []string) ([]*unit.SourceUnit, error) {
+func isInGopath(path string) bool {
+	for _, gopath := range filepath.SplitList(buildContext.GOPATH) {
+		if strings.HasPrefix(evalSymlinks(path), filepath.Join(evalSymlinks(gopath), "src")) {
```

Comment:
`filepath.HasPrefix` is deprecated and should not be used.

https://godoc.org/path/filepath#HasPrefix

> HasPrefix exists for historical compatibility and should not be used.